paparazzi/src/main/java/app/cash/paparazzi/internal/resources/RepositoryLoader.kt

```diff
@@ -365,14 +354,14 @@ abstract class RepositoryLoader<T : LoadableResourceRepository>(
           }
         } while (event != XmlPullParser.END_DOCUMENT)
       }
-    } // KXmlParser throws RuntimeException for an undefined prefix and an illegal attribute name.
-    catch (e: IOException) {
+    } catch (e: IOException) {
       handleParsingError(file, e)
     } catch (e: XmlPullParserException) {
       handleParsingError(file, e)
     } catch (e: XmlSyntaxException) {
       handleParsingError(file, e)
     } catch (e: RuntimeException) {
+      // KXmlParser throws RuntimeException for an undefined prefix and an illegal attribute name.
```

Comment:
Note: this was originally meant for a group of 4 catches, but I guess based on the wording it belongs here.

https://android.googlesource.com/platform/tools/base/+/6c21fa43c56804ab5f4d3a3fc59b5c1158750fa6/resource-repository/main/java/com/android/resources/base/RepositoryLoader.java#545

Reply:
obsoleted by suggestion to add:

```
ktlint_standard_discouraged-comment-location = disabled
```